### PR TITLE
217 connect signup

### PR DESCRIPTION
**Pull request by Shushunya**
Added POST axios request to onSubmit method. Error handling is another task so it is not implemented properly.

### CODE REVIEW

File: FrontEnd/package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "react-router-dom": "^6.4.1",
     "react-router-hash-link": "^2.4.3",
     "react-scripts": "5.0.1",
+    "swr": "^2.2.2",
```
Comment (popovycholeg) [inline FrontEnd/package.json:20]: Do you use it?

File: FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js [hunk 2]
```diff
@@ -61,13 +62,33 @@ export function SignUpFormContentComponent(props) {
...
+        is_startup: isChecked.startup,
+      },
+    };
+    
+    axios({
+      method: 'post',
+      url: `http://localhost:8000/api/auth/users/`,
+      withCredentials: false,
+      data: dataToSend
+    }).then(res => console.log(res.data)).catch(error => console.log(error))
```
Comment (popovycholeg) [inline FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js:89]: Should we do smth then after registration? I know that API doesn't return a token, but should we redirect the user to the logic page?

Comment (Shushunya) [reply FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js:89]: I suppose there's another task to add modal window. But there are some inconsistences so I left it for the next task.

File: FrontEnd/package.json (hunk 1 above)
Comment (Shushunya) [reply FrontEnd/package.json]: Now it's not in my PR. Thanks for comment :)

File: FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js (hunk 2 above)
Comment (Shushunya) [reply FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js:89]: I left ToDos for the future work.

**Review by Lvyshnevska — APPROVED** [review]

**Pull request merged**